src/app/sb-admin/components/pages/sb-user/sbuser.component.ts

```diff
@@ -87,5 +101,22 @@ export class SbUserComponent implements OnInit {
   onGlobalFilter(table: Table, event: Event) {
     table.filterGlobal((event.target as HTMLInputElement).value, 'contains');
   }
+  addNewUser() {
+    const ref = this.dialogService.open(AddEditUserComponent, { header: this.i18nextPipe.transform('USER_CREATE'), width: '30%', height: 'auto' });
```

Comment:
All this hardcoded data should declare as a constants in top section of the ts file